backend/src/providers/transactions.ts

```diff
@@ -10,6 +10,8 @@ import {
 import { z } from "zod";
 import { validators } from "../router/validators";
 
+import { sendJsonRpc } from "../utils/near";
```

Comment:
We use `* as nearApi` everywhere.
That's not a strict rule so far, but it would be nice to have things consistent